src/patternfly/components/SimpleList/simple-list.scss

```diff
@@ -1,38 +1,37 @@
 // @debug $simple-list; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$simple-list} {
+:root, .#{$simple-list}__item-link {
```

Comment:
@mcoker was there some styling issue that this solves? I'm not noticing any difference when removing the item-link selector from here, and just wondering whether we'd want to have CSS vars that aren't related to the item-link being defined within its scope?